src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -24,16 +26,29 @@ export default hh(class WorkspaceDashboard extends Component {
 
 
   render() {
-    const { workspace, failure } = this.state
+    const { workspace, failure, modal } = this.state
 
     return _.isEmpty(workspace) ?
       failure ?
         `Couldn't load workspace details: ${failure}` :
         spinner({ style: { marginTop: '1rem' } }) :
       div({ style: { margin: '1rem' } }, [
+        modal ? Modal({
+          onDismiss: () => this.setState({ modal: false }),
+          title: 'Workspace Info',
+          showCancel: false,
+          okButton: buttonPrimary({ onClick: () => this.setState({ modal: false }) }, 'Done')
+        }, [
+          div({ style: { whiteSpace: 'pre', overflow: 'auto', padding: '1rem' } },
+            JSON.stringify(workspace, null, 2))
```

Comment:
Great example! I see this minor issue: https://www.useloom.com/share/e32a7a430cf4457e8becc43ba9e5ca77 Feel free to add an issue to address in a follow-up if this is blocking other work.

Reply:
Created #60